boards/x86/acrn/acrn.dts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <mem.h>
+
+#define DT_FLASH_SIZE	DT_SIZE_K(8192)
```

Comment:
we're not running from flash, so I think we can set this to 0, this should really only apply to systems with XIP.

Reply:
Again, this is sort of an artifact of the existing infrastructure. See ia32.dtsi.